dist/obs-server.spec

```diff
@@ -188,7 +188,6 @@ BuildRequires: obs-server-macros
 
 %if 0%{?suse_version:1}
 BuildRequires:  fdupes
-PreReq:         %insserv_prereq permissions pwdutils
```

Comment:
actually, yes "permissions" can go as well, as "%run_permissions" call  was removed in commit d0c101da0986cd4d9df6e0f9754cf202e4f882bb from 2013 ...
but then the cleanup needs to be completed by removing

 %verifyscript -n obs-server
 %verify_permissions

as well ...

and "pwdutils" should be kept, but replaced by "shadow" as it's still used here:
%triggerin -n obs-server -- docker
usermod -a -G docker obsservicerun

so this should IMHO be:
-PreReq:         %insserv_prereq permissions pwdutils
+Requires(pre): shadow

Reply:
Thanks! `%verify*` lines removed, and suggested patch applied.